portability-transfer/src/main/java/org/dataportabilityproject/transfer/PortabilityInMemoryDataCopier.java

```diff
@@ -86,31 +90,45 @@ private void copyHelper(
       UUID jobId,
       AuthData exportAuthData,
       AuthData importAuthData,
-      Optional<ExportInformation> exportInformation)
-      throws IOException {
+      Optional<ExportInformation> exportInformation) {
 
     logger.debug("copy iteration: {}", COPY_ITERATION_COUNTER.incrementAndGet());
 
+    // TODO: read in retry strategies from a config, but that's for later in v1
+    RetryStrategy expBackoffStrategy = new ExponentialBackoffRetryStrategy(5, 10, 2);
+    RetryStrategyLibrary library = new RetryStrategyLibrary(new LinkedList<>(), expBackoffStrategy);
+
     // NOTE: order is important below, do the import of all the items, then do continuation
     // then do sub resources, this ensures all parents are populated before children get
     // processed.
     logger.debug("Starting export");
-    ExportResult<?> exportResult = export(jobId, exportAuthData, exportInformation);
-    logger.debug("Finished export");
-
-    if (exportResult.getType().equals(ResultType.ERROR)) {
-      logger.warn("Error happened during export: {}", exportResult.getMessage());
+    CallableExporter callableExporter = new CallableExporter(exporter, jobId, exportAuthData,
+        exportInformation);
+    RetryingCallable<ExportResult> exportRetryingCallable = new RetryingCallable<>(callableExporter,
+        library,
+        Clock.systemUTC());
+    ExportResult<?> exportResult;
+    try {
+      exportResult = exportRetryingCallable.call();
+    } catch (RetryException e) {
+      logger.warn("Error happened during export: {}", e);
       return;
     }
+    logger.debug("Finished export");
```

Comment:
If we aggregate these logs and don't look up individual worker logs one by one, it would be much more useful to include the job ID in all log statements

Reply:
Done.